manifests/init.pp

```diff
@@ -129,28 +126,27 @@
 #   Determines whether to restart nomad agent on $config_hash changes. This will not affect reloads when service, check or watch configs change.
 class nomad (
   String[1] $arch,
+  Stdlib::Absolutepath $bin_dir                  = '/usr/bin',
```

Comment:
Nit: I'd keep it on the same line as before for a smaller diff.

Reply:
moved it back into place